lib/cogctl/action_util.ex

```diff
@@ -78,12 +78,21 @@ defmodule Cogctl.ActionUtil do
     :ok
   end
 
-  def display_error(errors) when is_list(errors) do 
-    IO.puts(:stderr, "ERROR: " <> Enum.join(errors, "\n"))
+  def display_warning(warnings) when is_list(warnings) do
+    Enum.map(warnings, &display_warning/1)
```

Comment:
Just a heads up, but multiple `IO.puts` calls for the same command output can cause the tests to do weird things. We should probably be building up a list here and to `puts` all at once.